hypertrace-core-graphql-span-schema/src/main/java/org/hypertrace/core/graphql/span/dao/SpanLogEventRequestBuilder.java

```diff
@@ -29,31 +33,67 @@
   private final Converter<Collection<AttributeRequest>, Set<Expression>> attributeConverter;
   private final Converter<Collection<AttributeAssociation<FilterArgument>>, Filter> filterConverter;
   private final FilterRequestBuilder filterRequestBuilder;
+  private final AttributeStore attributeStore;
 
   @Inject
   SpanLogEventRequestBuilder(
       Converter<Collection<AttributeRequest>, Set<Expression>> attributeConverter,
       Converter<Collection<AttributeAssociation<FilterArgument>>, Filter> filterConverter,
-      FilterRequestBuilder filterRequestBuilder) {
+      FilterRequestBuilder filterRequestBuilder,
+      AttributeStore attributeStore) {
     this.attributeConverter = attributeConverter;
     this.filterConverter = filterConverter;
     this.filterRequestBuilder = filterRequestBuilder;
+    this.attributeStore = attributeStore;
   }
 
   Single<LogEventsRequest> buildLogEventsRequest(
       SpanRequest gqlRequest, SpansResponse spansResponse) {
-    return zip(
-        this.attributeConverter.convert(gqlRequest.logEventAttributes()),
-        buildLogEventsQueryFilter(gqlRequest, spansResponse).flatMap(filterConverter::convert),
-        (selections, filter) ->
-            LogEventsRequest.newBuilder()
-                .setStartTimeMillis(
-                    gqlRequest.spanEventsRequest().timeRange().startTime().toEpochMilli())
-                .setEndTimeMillis(
-                    gqlRequest.spanEventsRequest().timeRange().endTime().toEpochMilli())
-                .addAllSelection(selections)
-                .setFilter(filter)
-                .build());
+    return getRequestAttributes(
+            gqlRequest.spanEventsRequest().context(), gqlRequest.logEventAttributes())
+        .flatMap(
+            v ->
```

Comment:
Can we please have a more descriptive name for `v`? It could even say `attribute` or `requestAttribute` or whatever best describes it

Reply:
Renamed `v` at other places as well, will be careful next time on!